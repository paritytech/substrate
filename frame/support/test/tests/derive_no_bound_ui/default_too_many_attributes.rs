trait Config {
	type C;
}

#[derive(frame_support::DefaultNoBound)]
enum Foo<T: Config> {
	#[default]
	Bar(T::C),
	#[default]
	Baz,
}

fn main() {}
