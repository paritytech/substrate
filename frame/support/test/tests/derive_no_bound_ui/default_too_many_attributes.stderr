error: multiple declared defaults
 --> tests/derive_no_bound_ui/default_too_many_attributes.rs:5:10
  |
5 | #[derive(frame_support::DefaultNoBound)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `frame_support::DefaultNoBound` (in Nightly builds, run with -Z macro-backtrace for more info)

error: first default
 --> tests/derive_no_bound_ui/default_too_many_attributes.rs:7:2
  |
7 | /     #[default]
8 | |     Bar(T::C),
  | |_____________^

error: additional default
  --> tests/derive_no_bound_ui/default_too_many_attributes.rs:9:2
   |
9  | /     #[default]
10 | |     Baz,
   | |_______^
